test/_test.py

```diff
@@ -0,0 +1,39 @@
+import os
+from contextlib import contextmanager
+
+import common_utils
+import torchaudio
+
+BACKENDS = torchaudio._backend._audio_backends
+
+
+@contextmanager
+def AudioBackendScope(new_backend):
+    previous_backend = torchaudio.get_audio_backend()
+    try:
+        torchaudio.set_audio_backend(new_backend)
+        yield
+    finally:
+        torchaudio.set_audio_backend(previous_backend)
+
+
+def get_backends_with_mp3(backends):
```

Comment:
The purpose of this function was not immediately clear to me at the first time reading it. Adding docstring would be helpful.

Reply:
Thanks for pointing this out!